src/hadoop-ai/build/build.sh

```diff
@@ -32,6 +32,8 @@ git checkout branch-2.9.0
 git apply /hadoop-2.9.0.gpu-port.patch
 git apply /HDFS-13773.patch
 git apply /docker-executor.patch
+# to avoid potential endless loop, refer to https://issues.apache.org/jira/browse/YARN-8513?page=com.atlassian.jira.plugin.system.issuetabpanels%3Aall-tabpanel
+git apply /12944563.patch
```

Comment:
why not use the orignal Patch name, i.e. YARN-8896.patch

Reply:
It's not 2.9 patch